samples/interactive-tutorials/README.md

```diff
@@ -100,55 +127,69 @@ To run a code sample from the Cloud Shell, you need to be authenticated using th
 To run Python code samples for the Retail API tutorial, you need to set up your virtual environment.
 
 1. Run the following commands in a Terminal to create an isolated Python environment:
+
     ```bash
-    pip install virtualenv
-    virtualenv myenv
+    virtualenv -p python3 myenv
```

Comment:
Virtualenv is installed by default on python3:

https://docs.python.org/3/library/venv.html#creating-virtual-environments

```
python3 -m venv myenv
```

Reply:
we need to is because the CloudShell by default still has the python 2.7